cassiopeia/plotting/utilities.py

```diff
@@ -0,0 +1,525 @@
+"""
+Utility functions for use when plotting trees. Functions here support "local"
+plotting, as opposed to itol_utilities, which are for visualization through
+iTOL.
+"""
+from typing import Dict, List, Optional, Tuple, Union
+
+import networkx as nx
+import numpy as np
+import pandas as pd
+from matplotlib.colors import rgb_to_hsv
+from typing_extensions import Literal
+
+from cassiopeia.data import CassiopeiaTree
+from cassiopeia.preprocess import utilities as preprocess_utilities
+
+
+def degrees_to_radians(degrees) -> float:
+    """Convert degrees to radians.
+
+    Args:
+        degrees: Degrees
+
+    Returns:
+        Degrees converted to radians.
+    """
+    return degrees * np.pi / 180
+
+
+def polar_to_cartesian(degrees: float, r: float) -> Tuple[float, float]:
+    """Convert polar coordinates to cartesian coordinates.
+
+    Args:
+        degrees: The angle (in degrees)
+        r: The radius
+
+    Returns:
+        A tuple of x and y coordinates
+    """
+    theta = degrees_to_radians(degrees)
+    return r * np.cos(theta), r * np.sin(theta)
+
+
+def polars_to_cartesians(
+    degrees: List[float], rs: List[float]
```

Comment:
Would it make more sense to pass in a list of `Tuples`?

Reply:
I think the way it is currently makes the function easier to call, since all the coordinates are currently grouped by degrees and radians as separate lists. If we were to change this to take a list of tuples, every time we call this function we would have to transform the list of degrees and list of radians to a single list of tuples. I'll just leave this as is for now.